config/application.example.yml

```diff
@@ -1,3 +1,21 @@
 mandrill_api_key: "api_key"
 mandrill_username: "username"
-sentry_dsn: "api_key"
\ No newline at end of file
+sentry_dsn: "api_key"
+
+
+S3_BUCKET_NAME: "bucket_name"
+AWS_ACCESS_KEY_ID: "AJSDLKJASLKDJAUASD"
+AWS_SECRET_ACCESS_KEY: "asjdllaDSADJLASJASDSALDJALSDJLlkasjdlksad123123123"
```

Comment:
OMG you gave me a panic attack. Then I realized this was all fake. I suggest instead of random garbage here you put something like `YOUR_ACCESS_KEY_HERE` Remember EVERYTHING should have a point, EVERYTHING should justify its inclusion in the file.